Drift/Kernel.php

```diff
@@ -29,6 +29,8 @@ class Kernel extends AsyncKernel
 {
     use MicroKernelTrait;
 
+    private const CONFIG_EXTS = '.{php,xml,yaml,yml}';
```

Comment:
- `yaml` === `yml`, so I'd go for only `yml`.
- Almost no one configure services using PHP.
- I'm an XML hater, and as a user and a programmer, I think that almost everyone understands YML at first sight, while in XML you must make an extra effort to split between the "struicture definition" part and the content part.

Reply:
Looks like Symfony is going to use PHP configs in the future. 🤔 
https://github.com/symfony/symfony/issues/37186